docker-compose.override.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
Yeah, it's automatically used by docker-compose.yml when not providing explicit docker-compose config files: https://docs.docker.com/compose/extends/#/understanding-multiple-compose-files